instagram/src/components/Authentication/Authenticate.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import Login from "../Login/Login";
+
+const authenticate = WrappedComponent => {
```

Comment:
Since this is still _technically_ a component, you should still capitalize it (`Authenticate`)